### PR TITLE
pathd8 1.0 (new formula)

### PR DESCRIPTION
**Pull request by jonchang**
PATHd8 performs phylogenetic dating of large trees without a molecular clock.

### CODE REVIEW

**Comment by tseemann** [comment]
@sjackman @iml Should the ```cc``` be changed to ```ENV.cc``` ?

**Comment by iml** [comment]
Yes, `system ENV.cc` etc.

File: pathd8.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  # tag "bioinformatics"
+
+  def install
+    # Build instructions per http://www2.math.su.se/PATHd8/compile.html
+    system "cc", "PATHd8.c", "-O3", "-lm", "-o", "PATHd8"
+    bin.install "PATHd8"
+  end
+
+  test do
+    cd testpath do
```
Comment (iml) [inline pathd8.rb]: You can remove the `cd` it's already implied inside the test block.

**Comment by tseemann** [comment]
@jonchang If you can make the above changes and squash into a single commit that would be very helpful. We appreciate the contribution.

**Comment by sjackman** [comment]
@tseemann If you plan on taking a PR/issue, feel free to assign the issue to yourself. If you just want to fly by with a few comments on a strafing run (as I often do), leave it unassigned.

**Comment by jonchang** [comment]
Fixed and pushed.

**Comment by iml** [comment]
@BrewTestBot test this please.

**Comment by iml** [comment]
Thanks @jonchang!

**Pull request closed without merging**